changelogs/fragments/273-Dont_call_get_on_None

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
The description needs to start with a lowercase letter, so, change the leading `If` to `if`.  To write code, I believe you want `C(None)` or `C(dict)` or `C(get())`, versus wrapping in double backticks, [here's](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html) what I'm reading. Lastly, this file needs either the `yml` or `yaml` file extension.

Reply:
i was referencing [this](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment), which is specific to writing the changelog fragment, when i used double ticks for code.